### PR TITLE
Bump tuf-test-vectors and switch to uptane namespace.

### PR DESCRIPTION
**Pull request by pattivacek**
Split out from https://github.com/uptane/aktualizr/pull/65 since this has been surprisingly error-prone.

### CODE REVIEW

**Comment by pattivacek** [comment]
Wow, this took forever to sort out. I finally fixed it. From the last commit:

```
Manually fetch the submodules in github CI.
    
    The automatic fetching logic breaks in really confusing ways when the
    submodule origin has changed. It really doesn't make sense. This is a
    really naïve workaround. It is hopefully just a temporary fix that can
    be reverted at a later time, but at least for now, it seems to be
    necessary in order to get CI to pass on this branch.
```

**Comment by mike-sul** [comment]
> It is hopefully just a temporary fix that can
>     be reverted at a later time, but at least for now, it seems to be
>     necessary in order to get CI to pass on this branch.

It's really odd indeed. I think the workaround can be reverted too.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -92,10 +92,10 @@ jobs:
         -eTEST_WITH_STATICTESTS=1
         -eTEST_WITH_DOCS=1
     steps:
-      - uses: actions/checkout@master
-        with:
-          submodules: recursive
-      - run: git fetch --prune --unshallow
+      - uses: actions/checkout@main
```
Comment (mike-sul) [inline .github/workflows/ci.yml:95]: Why is it `main` branch if this repo doesn't have such?

Comment (mike-sul) [reply .github/workflows/ci.yml:95]: oh, this is must be `action` itself

**Review by mike-sul — APPROVED** [review]

**Pull request merged**